### PR TITLE
Change Window Title

### PR DESCRIPTION
**Pull request by Affehund**
This PR adds a mixin to change the Window Title
Bildschirmfoto 2022-08-29 um 11 30 24
Bildschirmfoto 2022-08-29 um 11 35 50

### CODE REVIEW

**Comment by marios-andr** [comment]
I believe that the title should be something like "Minecraft Brass" or "Minecraft // Brass" instead. Since the main game is Minecraft, I don't think it should be abbreviated.

File: src/mixin/java/io/github/brassmc/brassloader/mixin/MinecraftMixin.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 @Mixin(Minecraft.class)
 public class MinecraftMixin {
+
```
Comment (Matyrobbrt) [inline src/mixin/java/io/github/brassmc/brassloader/mixin/MinecraftMixin.java]:
```suggestion
```

**Review by Matyrobbrt — CHANGES_REQUESTED** [review]
I'm not precisely sure if the modded indicator should be removed, I'd advocate for keeping it.

**Comment by Affehund** [comment]
> I'm not precisely sure if the modded indicator should be removed, I'd advocate for keeping it.

I think it just looks much cleaner if we remove the * indicator, but we could keep it. 
And I think it is clear that it is modded as it there is a different name.

**Comment by marios-andr** [comment]
> > I'm not precisely sure if the modded indicator should be removed, I'd advocate for keeping it.
> 
> I think it just looks much cleaner if we remove the * indicator, but we could keep it. And I think it is clear that it is modded as it there is a different name.

I agree, * is used to indicate the client is modded, so I think the whole "Minecraft Brass" does that very well

**Comment by Affehund** [comment]
> > > I'm not precisely sure if the modded indicator should be removed, I'd advocate for keeping it.
> > 
> > 
> > I think it just looks much cleaner if we remove the * indicator, but we could keep it. And I think it is clear that it is modded as it there is a different name.
> 
> I agree, * is used to indicate the client is modded, so I think the whole "Minecraft Brass" does that very well

Ok, then I would leave it how it is, unless someone is strictly against it.

**Review by DaRealTurtyWurty — CHANGES_REQUESTED** [review]
I think the `*` should really be present here. Its more of a standardization than anything else, not about whether it makes sense or not. People very quickly associate `*` with being modded, so I don't think it makes sense to ignore that standard.

**Review by Matyrobbrt — APPROVED** [review]

**Pull request merged**